packages/bpk-component-overlay/README.md

```diff
@@ -45,5 +44,4 @@ export default () => (
 | children | Node | true | - |
 | className | string | false | null |
 | foregroundContent | Node | false | null |
-| overlayType | oneOf(OVERLAY_TYPES) | false | OVERLAY_TYPES.solid |
-| overlayLevel | oneOf(OVERLAY_LEVELS) | false | OVERLAY_LEVELS.low |
+| overlayType | oneOf(OVERLAY_TYPES) | false | OVERLAY_TYPES.solidHigh |
```

Comment:
the default value is actually `solidLow` 🙂

Reply:
That's right, will change it now:)